setup.py

```diff
@@ -10,7 +10,7 @@
                       'plyvel', 'pylru', 'aiohttp>=3.3,<4'],
     extras_require={
         'rapidjson': ['python-rapidjson>=0.4.1,<2.0'],
-        'rocksdb': ['python-rocksdb>=0.6.9'],
+        'rocksdb': ['python-rocksdb-static'],
```

Comment:
why? this affects bitcoin users too, so needs proper justification

Reply:
That's the only way I could get it to be installed. Does the python-rocksdb package still work for you?
https://github.com/twmht/python-rocksdb/issues/102

Reply:
well it [works for the CI](https://cirrus-ci.com/task/4922075872755712) on current master.
but I guess it might be compiling against older rocksdb

Nevertheless, please open a separate PR (or issue) for the rocksdb change. It does not belong in this PR.